lib/FusionInventory/Agent/Task/Inventory/Generic/Ipmi/Fru.pm

```diff
@@ -0,0 +1,55 @@
+#
```

Comment:
Don't copy this copyright block here, it has nothing to do there as you just got it from old IPMI.pm module